index.js

```diff
@@ -14,11 +14,10 @@
  * Module dependencies.
  */
 
-var os = require('os');
+var osName = require('os-name');
 
-var USER_AGENT = 'node/' + process.version
-  + ' (' + process.platform + ' ' + os.release()
-  + '; ' + process.arch + ')';
+var USER_AGENT = 'Node.js/' + process.version.slice(1)
```

Comment:
How about using `process.release` https://github.com/iojs/io.js/pull/493 to support io.js also?